ui/app/templates/components/transit-key-action/verify.hbs

```diff
@@ -189,11 +189,14 @@
 </form>
 <Modal @title="Results" @onClose={{action (mut @isModalActive) false}} @isActive={{@isModalActive}}>
   <section class="modal-card-body">
-    <AlertBanner
-      @type={{if @valid "success" "danger"}}
-      @title={{if @valid "Valid" "Not Valid"}}
-      @message="The input is {{if @valid 'valid' 'not valid'}} for the given {{if @signature 'signature' 'HMAC'}}"
-      data-test-transit-verify="true"
-    />
+    <Hds::Alert @type="inline" @color={{if @valid "success" "critical"}} as |A|>
+      <A.Title>{{if @valid "Valid" "Not Valid"}}</A.Title>
+      <A.Description>
+        The input is
+        {{if @valid "valid" "not valid"}}
+        for the given
+        {{if @signature "signature" "HMAC"}}
```

Comment:
I believe, because this is a description, both signature and HMAC need a period?